open-reac/src/main/java/com/powsybl/openreac/parameters/output/network/AbstractNetworkOutput.java

```diff
@@ -43,25 +42,23 @@ public String getFileName() {
     }
 
     @Override
-    public void read(Path path, StringToIntMapper<AmplSubset> stringToIntMapper) {
-        List<String> lines = null;
-        try {
-            lines = Files.readAllLines(path);
-        } catch (IOException e) {
```

Comment:
There is a little difference here.

Previously, no exception was thrown when an `IOException` was thrown during `readAllLines` .

Now, no exception is thrown when the file is not a regular file (real file + read access on it), but if another `IOException` is thrown, it will be propagated as an `UncheckedIOException` by the caller (see `AmplModelExecutionHandler.readCustomFiles` in powsybl-core). But I don't know if it can really happen.